starter-core/src/test/groovy/io/micronaut/starter/feature/lang/groovy/module/GroovyModuleFeatureSpec.groovy

```diff
@@ -0,0 +1,79 @@
+package io.micronaut.starter.feature.lang.groovy.module
+
+import io.micronaut.starter.ApplicationContextSpec
+import io.micronaut.starter.BuildBuilder
+import io.micronaut.starter.build.BuildTestUtil
+import io.micronaut.starter.build.BuildTestVerifier
+import io.micronaut.starter.build.dependencies.Scope
+import io.micronaut.starter.feature.Category
+import io.micronaut.starter.fixture.CommandOutputFixture
+import io.micronaut.starter.options.BuildTool
+import io.micronaut.starter.options.Language
+import io.micronaut.starter.options.TestFramework
+
+class GroovyModuleFeatureSpec extends ApplicationContextSpec implements CommandOutputFixture {
+
+    void "test feature #moduleFeature with build tool #buildTool and Groovy language"(String moduleFeature, BuildTool buildTool) {
+        when:
+        String template = new BuildBuilder(beanContext, buildTool)
+                .language(Language.GROOVY)
+                .features([moduleFeature])
+                .render()
+        BuildTestVerifier verifier = BuildTestUtil.verifier(buildTool, template)
+
+        then:
+        verifier.hasDependency("org.apache.groovy", moduleFeature, Scope.TEST)
```

Comment:
If it's a Groovy application, shouldn't this be an implementation dependency, not a `TEST` one?

Reply:
The issue and @sdelamo otherwise both suggest adding them to the test classpath.

Reply:
I have no preference/objection one way or the other. Sergio suggested it in a private discussion, "probably adding those dependencies to the test classpath". We can ask when he's back.

Reply:
@timyates I changed to what I think is the correct compromise: if language is Groovy, add to compile classpath; otherwise, if test framework is Spock add to test classpath.